### PR TITLE
[RTI-14146] Update repo to Nextroll standards (plugins, CI, etc)

### PR DESCRIPTION
**Pull request by pablocostass**

### CODE REVIEW

File: src/rebar3_gpb_compiler.erl [hunk 1]
```diff
@@ -157,48 +152,47 @@ get_dependencies(Proto, Protos, GpbOpts) ->
...
-
 is_wanted_proto(WantedProto, ProtoPath) ->
     case string:sub_string(ProtoPath, length(ProtoPath) - length(WantedProto) + 1) of
-        WantedProto -> true;
-        _Other -> false
+        WantedProto ->
+            true;
+        _Other ->
+            false
     end.
```
Comment (elbrujohalcon) [inline src/rebar3_gpb_compiler.erl]:
```suggestion
    WantedProto == string:sub_string(ProtoPath, length(ProtoPath) - length(WantedProto) + 1).
```

File: src/rebar3_gpb_prv_compile.erl [hunk 2]
```diff
@@ -1,54 +1,53 @@
...
+        providers:create([{name, ?PROVIDER},            % The 'user friendly' name of the task
+                          {namespace, protobuf},
+                          {module, ?MODULE},            % The module implementation of the task
+                          {bare,
+                           true},                 % The task can be run by the user, always true
+                          {deps, ?DEPS},                % The list of dependencies
+                          {opts, []},                   % list of options understood by the plugin
+                          {example, "rebar3 protobuf compile"},
+                          {short_desc, ?SHORT_DESC},
+                          {desc, ?DESC}]),
```
Comment (elbrujohalcon) [inline src/rebar3_gpb_prv_compile.erl]: Do you mind moving the comments around a bit to avoid strange stuff like `{bare, true}` in two lines?

**Review by elbrujohalcon — CHANGES_REQUESTED** [review]

File: src/rebar3_gpb_compiler.erl (hunk 1 above)
Comment (pablocostass) [reply src/rebar3_gpb_compiler.erl]: Weirdly enough Github doesn't allow me to commit the suggestion from the UI; I will include this change on the next commit :)

**Comment by pablocostass** [comment]
I appreciate the previous comments, but I would suggest abstaining from changing more code unless necessary; the project has no tests 😱

**Review by elbrujohalcon — APPROVED** [review]
I'm approving, @pablocostass… but add a ticket to JIRA to add tests to this library.

**Comment by pablocostass** [comment]
Merging with one approval as the changes are mostly rebar3 or CI related.

**Pull request merged**